src/Observability/Runtime/Common/PowerPlatformApiDiscovery.cs

```diff
@@ -13,14 +13,17 @@ namespace Microsoft.Agents.A365.Observability.Runtime.Common
     public class PowerPlatformApiDiscovery
```

Comment:
There shouldn't be any changes in this class. The domain override environment variable needs to be the fully formed tenant island domain.